frontend/projects/setup-wizard/src/app/app.component.ts

```diff
@@ -29,7 +29,7 @@ export class AppComponent {
         this.stateService.isMigrating = false
         await this.navCtrl.navigateForward(`/recover`)
       }
-    } catch (e) {
+    } catch (e: any) {
```

Comment:
Why are these `any` and not the error type (eg. HttpError, RpcError, EncryptionError)

Reply:
Ok, I think unknown is more appropriate in this case.